Packages/ca.tekly.common/Runtime/Utils/UnityRuntimeEditorUtils.cs

```diff
@@ -1,45 +1,77 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Diagnostics;
+using UnityEditor;
 using Debug = UnityEngine.Debug;
 
 namespace Tekly.Common.Utils
 {
     public static class UnityRuntimeEditorUtils
     {
-        private static readonly List<Action> s_onExitActions = new List<Action>();
-        
+        private static readonly List<Action> s_onEnterActions = new();
```

Comment:
I've been staying away from using `new();` to make things backwards compatible with older versions of Unity. I'm not sure if I've caught all the ones I've added myself yet.